packages/adaptivecards-tools-sdk/rollup.config.js

```diff
@@ -33,5 +34,6 @@ export default {
     cjs(),
     typescript(),
```

Comment:
Leveraging typescript plugin would be easier. You could specify the option here:
```typescript
typescript({
  useTsconfigDeclarationDir: true,
}),
```

Reply:
This is helpful, thanks!